test/import-glob.js

```diff
@@ -89,33 +100,33 @@ test('does not transform standard import', t => {
 
 test('constructs the member by identifierfying the file name, without the common extname', t => {
   t.is(
-    transform("import { fooBar } from 'glob:fixtures/foo-bar/*.txt'"),
+    transform("import { fooBar } from 'glob:./fixtures/foo-bar/*.txt'"),
     "import fooBar from './fixtures/foo-bar/foo-bar.txt';")
 })
 
 test('constructs the member by identifierfying the file name, including remaining extnames', t => {
```

Comment:
Not sure. I just maintained the existing tests. If the branch coverage stays and same and you feel they are not needed I'll remove them.